test/sharness/t0060-daemon.sh

```diff
@@ -121,14 +121,30 @@ test_expect_success "daemon with pipe eventually becomes live" '
   test_fsh cat stdin_daemon_out || test_fsh cat stdin_daemon_err || test_fsh cat stdin_poll_apiout || test_fsh cat stdin_poll_apierr
 '
 
-ulimit -n 512
+ulimit -S -n 512
 TEST_ULIMIT_PRESET=1
 test_launch_ipfs_daemon
 
 test_expect_success "daemon raised its fd limit" '
-	grep "ulimit" actual_daemon > /dev/null
+	grep "raised file descriptor limit to 1024." actual_daemon > /dev/null
 '
 
+get_col_four() {
+	awk '{ print $4 }' $1
+}
+
+if [ `uname` == "Linux" ]; then
```

Comment:
- this is nice
- but we really need to also test in osx. and we shouldn't trust what the limits say, should test for sure that we can indeed open that many fds. 
- http://stackoverflow.com/questions/31115764/setrlimit-doesnt-work-on-os-x-maverick-and-yosemite
- we could do this with a tool like `open-fds <fds> <multiaddr>` which uses `nc` to hang fds open.
- then count the fds with `lsof`. maybe: `lsof -p $1 | grep -v " txt " | wc -l`